### PR TITLE
Disk usage can take more than few seconds to be computed when you have many jobs/files .... cc @reviewbybees

### PR DESCRIPTION
**Pull request by aheritier**

### CODE REVIEW

**Comment by aheritier** [comment]
In fact this PR duplicates the one here : https://github.com/cloudbees/cloudbees-disk-usage-simple/pull/3
Where is the official repository @ndeloof ?

**Comment by aheritier** [comment]
please @reviewbybees

**Comment by volker-fr** [comment]
I would suggest to call it a few minutes. Even that might be too nice. We experienced over 10minutes.

**Comment by oleg-nenashev** [comment]
"the calculation may take several minutes" (hello to Microsoft)

File: src/main/resources/com/cloudbees/simplediskusage/QuickDiskUsage/index.jelly [hunk 1]
```diff
@@ -33,7 +33,7 @@
             <j:choose>
                 <p>
                 <j:when test="${it.lastRun == 0}">
-                    not yet calculated, please <a href=".">try again in few seconds</a>
+                    not yet calculated, please <a href=".">try again in a few moments</a>
```
Comment (jtnord) [inline src/main/resources/com/cloudbees/simplediskusage/QuickDiskUsage/index.jelly]: a moment is a short period of time (usually seconds), o a distinct point in time.  
If you have multiple of them they will be disjoint - and as such I don't think this is correct :-)

"try again in a short while"? is possibly more what you are intending.

**Comment by jtnord** [comment]
:bug: as per inline - and others comments.

**Comment by aheritier** [comment]
Will be fixed in #4

**Comment by ndeloof** [comment]
@aheritier if you want to play with Jenkins API, a progressive rendering UI would be cool to show user already computed disk space per job

**Comment by aheritier** [comment]
thanks @ndeloof I'll have a look at this (when I'll have some time).

**Pull request closed without merging**